### ISSUE TITLE
[Adaptive Card iOS - Action.title.json]: Voice over does not convey the name for the button as name for the button is not defined under action. Title screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8292]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.2
Application: Adaptive Cards
Screen Reader: Voice Over
App Version: 1.0 (2.4.0-beta.23.1.8.1)

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'test' control and activate it.
6. Navigate to 'action.title.json' and activate it.
7. Navigate to the button and observe the issue.

**Actual Result:**
Name is not defined for the button due to which voice over simply announce as button.

**Expected Result:**
Name should be defined for the button so that voice over does convey the name for the button.
.
Note:
1. Same issue is repro under ColumnSet.Input.ChoiceSet.VerticalStretch.
2. This issue is repro throughout application.

**User Experience:**
It will impact the screen reader user if the name is not defined for the button.
 
**MAS Reference:**
[Name, Role, Value - Liquid (microsoft.com)][(https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02)](https://microsoft.sharepoint.com/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc={507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D}&file=MAS%204.1.2%20%E2%80%93%20Name,%20Role,%20Value.docx&action=default&mobileredirect=true&DefaultItemOpen=1)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/217247988-4134072b-0d0b-4a35-9abf-4fb00ba5bbb5.MP4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
@jwoo-msft Can we transfer/close this issue since we no longer own iOS?

**Comment by vagpt** [comment]
Closing this issue here, and we will log a new issue in new ADO if the same issue still persist at that time.
